packages/cactus-cmd-api-server/package.json

```diff
@@ -43,6 +43,7 @@
     "dist/*"
   ],
   "scripts": {
+    "benchmark": "tsx ./src/test/typescript/benchmark/run-cmd-api-server-benchmark.ts > ../../.tmp/benchmark-results/cmd-api-server/run-cmd-api-server-benchmark.ts.log",
     "codegen": "run-p 'codegen:*'",
```

Comment:
Fails if `../../.tmp/benchmark-results/cmd-api-server/` was not created beforehand. Maybe add `mkdir` command here before? Or don't pipe the output anywhere (print to stdout) and let the user decide where to store the results. CI can pipe it to location it created before.

Reply:
@outSH That was a rookie mistake on my side! Thanks for checking on it! 

I thought about just doing the quick and dirty `mkdir -p &&  ...` script but then the script wouldn't be able to run on windows (currently the project build doesn't work on windows anyway but at least I'm not getting us further away from it either)

So, I ended up refactoring the script in a way that it takes a path parameter and then does the mkdir -p from code which is cross-platform by default. I tested it by deleting the `.tmp` folder and then re-running the script and it works fine, please have a look when you get the chance!

Reply:
@petermetz Thanks, looks great!